### PR TITLE
Paywalls: Small fixes and examples for UIKit and modes

### PR DESCRIPTION
**Pull request by joshdholtz**
## Motivation / Description

Improving docs from questions and examples that were given in beta tester Discord

## Changes introduced

- Fixed a variable typo
- Added UIKit example to full screen paywall
- Added mode examples for SwiftUI and UIKit

## Jira ticket (if any)

N/A

## Additional comments

N/A

### CODE REVIEW

File: code_blocks/🚀 Getting Started/displaying-products/paywalls/paywalls_5.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        VStack {
+            ScrollView {
+                VStack {
+                    // Your custom paywall design content
+                }.frame(maxWidth: .infinity)
+            }
+
+            Spacer()
+
+            PaywallView(mode: .condensedCard) // or .card
```
Comment (NachoSoto) [inline code_blocks/🚀 Getting Started/displaying-products/paywalls/paywalls_5.swift:15]: <3

File: code_blocks/🚀 Getting Started/displaying-products/paywalls/paywalls_5.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import RevenueCatUI
```
Comment (NachoSoto) [inline code_blocks/🚀 Getting Started/displaying-products/paywalls/paywalls_5.swift:1]: Let's add `import SwiftUI`

File: code_blocks/🚀 Getting Started/displaying-products/paywalls/paywalls_4.swift [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import UIKit
+
```
Comment (NachoSoto) [inline code_blocks/🚀 Getting Started/displaying-products/paywalls/paywalls_4.swift:2]:
```suggestion
import RevenueCat
import RevenueCatUI
```

File: code_blocks/🚀 Getting Started/displaying-products/paywalls/paywalls_6.swift [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import UIKit
```
Comment (NachoSoto) [inline code_blocks/🚀 Getting Started/displaying-products/paywalls/paywalls_6.swift:1]:
```suggestion
import UIKit

import RevenueCatUI
import RevenueCat
```

**Review by NachoSoto — APPROVED** [review]
Yess 👏🏻

File: docs_source/🚀 Getting Started/displaying-products/paywalls.md [hunk 5]
```diff
@@ -156,6 +160,55 @@ Use your own hex codes, select a custom color, or use our color picker to select
...
+    "file": "code_blocks/🚀 Getting Started/displaying-products/paywalls/paywalls_4.swift"
+  }
+]
+[/block]
+
+## How to embed a Paywall into your views
+
+RevenueCatUI also has smaller paywalls for you to embed or overlay in your app. You could display these as inline banners in your app, overlay your app as a sheet promoting an upgrade, or as a footer in a custom paywall.
+
+This can all be done by passing `PaywallViewMode` into `PaywallView`. The options are:
```
Comment (NachoSoto) [inline docs_source/🚀 Getting Started/displaying-products/paywalls.md:180]: Maybe we can add a screenshiot here?

Comment (joshdholtz) [reply docs_source/🚀 Getting Started/displaying-products/paywalls.md:180]: Yeah yeah! I'll add some screenshots in a follow up PR tomorrow after working with @dpannasch on we want the sample screenshots to look like 💪  But I think that would totally helpful!

**Comment by joshdholtz** [comment]
@NachoSoto Added imports for either SwiftUI or UIKit to all the paywall examples!

**Pull request merged**